hatchery/tests/callcenter.rs

```diff
@@ -12,14 +12,14 @@ pub fn world_center_counter_read() -> Result<(), Error> {
 
     let counter_id = world.deploy(module_bytecode!("counter"))?;
 
-    let value: Receipt<i64> = world.query(counter_id, "read_value", ())?;
+    let value: Receipt<i64> = world.query(0, counter_id, "read_value", ())?;
```

Comment:
I'm not to happy with this being handled on every single query. We should instead make it a method on world.